duo_openvpn_as.py

```diff
@@ -204,20 +203,36 @@ def sign(ikey, skey, method, host, uri, date, sig_version, params):
     Return basic authorization header line with a Duo Web API signature.
```

Comment:
The comments I'm leaving for this function are essentially trying to get it to look more like
https://github.com/duosecurity/duo_client_python/blob/master/duo_client/client.py#L107
just wanted to help you see where I'm coming from. Hopefully this helps!

Reply:
yes, this helps.